File: branches/UVa/uva11727.cpp
/*Filename: uva11727.cpp
 *Author: Mushiyo
 */

#include <iostream>
#include <algorithm>

using namespace std;

int main() {
	const int EMPLOYEE_NUM = 3;
	int employee[EMPLOYEE_NUM];
	int T = 0;

	while (cin >> T) {
		for (int caseNum = 1; caseNum <= T; ++caseNum) {
			for (int i = 0; i < EMPLOYEE_NUM; ++i) {
				cin >> employee[i];
			}

			sort(employee, employee + EMPLOYEE_NUM);

			cout << "Case " << caseNum << ": " << employee[1] << endl;
		}
	}

	return 0;
}

File: trunk/UVa/UVa11727.java
import java.util.Scanner;
import java.util.Arrays;

public class UVa11727 {

	public static void main(String[] args) {
		final int EMPLOYEE_NUM = 3;
		int[] employee = new int[EMPLOYEE_NUM];
		
		Scanner input = new Scanner(System.in);
		
		while(input.hasNext()){
			int T = input.nextInt();
			
			for(int caseNum = 1; caseNum <= T; ++caseNum){
				for(int i = 0; i < EMPLOYEE_NUM; ++i){
					employee[i] = input.nextInt();
				}
				
				Arrays.sort(employee);
				
				System.out.println("Case " + caseNum + ": " + employee[1]);
			}
		}
	}

}
